libraries/tuttle/src/tuttle/plugin/context/WriterPlugin.cpp

```diff
@@ -32,7 +32,6 @@ WriterPlugin::WriterPlugin( OfxImageEffectHandle handle )
 	_paramPremult = fetchBooleanParam( kParamPremultiplied );
 	_paramExistingFile = fetchChoiceParam( kParamWriterExistingFile );
 	_paramForceNewRender = fetchIntParam( kParamWriterForceNewRender );
-	_isSequence = sequenceParser::browseSequence( _filePattern, _paramFilepath->getValue() );
```

Comment:
_isSequence should be initialized here or call:
`changedParam(kTuttlePluginFilename, OFX::InstanceChangedArgs());`

For example, if you load a nuke project file `changedParam` is not called and _isSequence will remain false.